### DISCUSSION TITLE
Why does my treasury show $0?

### DISCUSSION THREAD

**Discussion by mszulcz-mitre** [snapshot-labs/snapshot#3387] [Q&A] [answered]
I linked a Gnosis Safe treasury with funds in it to my Snapshot space, but the Treasury page for the space shows it has $0 in funds.  Why is that?  It seems the treasury is correctly linked because, when I click on the treasury address on the Treasury page, I'm taken to the correct etherscan.io page.

Does the Treasury page only report funds above a certain amount?  

Does the reporting of funds not work for the Goerli network?  The [Gnosis Safe page](https://docs.snapshot.org/gnosis-safe) in the docs indicates that using "Gnosis Safe to vote, create a proposal or setup a space on Snapshot" only works for Ethereum Mainnet, Optimism Mainnet and Polygon Mainnet, but the [docs on treasuries in general](https://docs.snapshot.org/spaces/settings#treasuries) don't indicate any limitations.

**Comment by samwhosung** [comment, marked as answer]
Tokens on Goerli network don't have a price because it's a testnet.

**Reply by mszulcz-mitre** [reply]
Thanks for your reply!  Is there a way to see the number of Goerli ETH?
